cmake/developer_package/ncc_naming_style/openvino.style

```diff
@@ -6,7 +6,7 @@ ClassName: '^([A-Z][\w]+|b?float16|numeric_limits|ngraph_error|stopwatch|unsuppo
 StructName: '^([A-Z][\w]+|element_type_traits|hash|oi_pair)$'
 FunctionName: '^(operator\W+|[a-z_\d]+)$'
 Namespace: '^([a-z\d_]+|InferenceEngine)$'
-NamespaceAlias: '^[a-z\d_]+$'
+NamespaceAlias: '^([a-z\d_]+|InferenceEngine)$'
```

Comment:
Do we need to have this change? You created an alias in common.hpp.

Reply:
But `common.hpp` is also checked by the `ncc` tool